apps/docs/src/content/docs/getting-started/usage.mdx

```diff
@@ -78,46 +78,7 @@ type Task = {
 
 ## `TaskInstance` structure
 
-- `error`: if an error is thrown inside the task instance, it will be found here
-- `isCanceled`: whether the task instance was canceled
-- `isError`: whether the task instance throw an error before completing
-- `isRunning`: whether the task instance is currently running
-- `isSuccessful`: whether the task instance completed successfully
-- `value`: if the task instance completed successfully, this will be the return value
-
-```ts
-type TaskInstance<TReturn> = {
-	/**
-	 * If an error is thrown inside the task instance, it will be found here.
-	 */
-	error?: Error;
-
-	/**
-	 * Indicates whether the task instance was canceled.
-	 */
-	isCanceled: boolean;
-
-	/**
-	 * Indicates whether the task instance threw an error before completing.
-	 */
-	isError: boolean;
-
-	/**
-	 * Indicates whether the task instance is currently running.
-	 */
-	isRunning: boolean;
-
-	/**
-	 * Indicates whether the task instance completed successfully.
-	 */
-	isSuccessful: boolean;
-
-	/**
-	 * If the task instance completed successfully, this will be the return value.
-	 */
-	value?: TReturn;
-};
-```
+To make it easier to reason about, every type of task modifier utilizes the same underlying structure. You can find more detail about the structure of the `TaskInstance` in the [reference docs](/reference/task-instance).
```

Comment:
I think we can use the Link component from Astro to have nice link boxes

Reply:
Can you point me towards this component, I can't see it anywhere. Also, we aren't using the Link component in any of the other reference docs - https://github.com/mainmatter/sheepdog/blob/b8bc4692bc4a7dd3ceeb22e0853e7dba5a13ffe8/apps/docs/src/content/docs/reference/default.mdx?plain=1#L181

Reply:
It's the `LinkCard` component, you can see usage in various places.

Reply:
I'm not a big fan of how this looks to be honest. What do you think?
![CleanShot 2024-09-27 at 13 20 07](https://github.com/user-attachments/assets/9584b0be-3d75-4ec7-9d9e-54bf7da70752)

Reply:
Mmm yeah I think you are right